rollup/internal/controller/relayer/l1_relayer.go

```diff
@@ -230,6 +234,20 @@ func (r *Layer1Relayer) StopSenders() {
 }
 
 func (r *Layer1Relayer) shouldUpdateGasOracle(baseFee uint64, blobBaseFee uint64, isCurie bool) bool {
+	fields := map[string]interface{}{
+		"rollup_status": types.RollupCommitting,
```

Comment:
Not clear what's the best solution here, but I think checking the update time for the latest `types.RollupCommitted` entry might be better. I.e. the last successful rollup tx that we sent to L1.

Also, we should consider updating the gas oracle in this case to some default value, otherwise it might get stuck at an expensive setting.

Reply:
Added default `baseFee` and `blobBaseFee` to configurations, but any suggestions what default value should be chosen?